Formula/lfortran.rb

```diff
@@ -0,0 +1,123 @@
+class Lfortran < Formula
+  include Language::Python::Virtualenv
+
+  desc "Modern interactive LLVM-based Fortran compiler"
+  homepage "https://lfortran.org/"
+  url "https://lfortran.github.io/tarballs/release/lfortran-0.2.0.tar.gz"
+  sha256 "729b38269b1c933f3b2dc9253e1fab7e7aec4206d518e9cf007b171da5af53c3"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@9"
+  depends_on "python"
```

Comment:
maybe `python@3.8`?

Reply:
Nope, LFortran links to Python 3.7 methods and gives compile errors with Python 3.8

Reply:
This is pointing to python@3.8